Libs/GObject-2.0/Native/Classes/TypeDictionary.cs

```diff
@@ -97,6 +97,9 @@ internal static System.Type GetSystemType(Type gtype)
             if (_reverseTypeDict.TryGetValue(gtype, out System.Type? sysType))
                 return sysType;
 
+            if (Functions.TypeIsA(gtype.Value, Type.Boxed.Value))
+                throw new Exception($"Record not registered: {gtype.ToString()}");
```

Comment:
This message is irritating. The message says that the record is not registred but the condition resulting in the error checks that boxed types are not supported in general? Either the condition is wrong or the message must be adjusted.

Reply:
Either way why do we need this at all?